qiskit/extensions/quantum_initializer/ucg.py

```diff
@@ -244,20 +245,20 @@ def _demultiplex_single_uc(self, a, b):
             d = np.flip(d, 0)
             u = np.flip(u, 1)
         d = np.diag(np.sqrt(d))
-        v = d.dot(_ct(u)).dot(_ct(r)).dot(b)
+        v = d.dot(UCG._ct(u)).dot(UCG._ct(r)).dot(b)
         return v, u, r
 
+    @staticmethod
+    def _ct(m):
+        return np.transpose(np.conjugate(m))
 
-def _ct(m):
-    return np.transpose(np.conjugate(m))
+    @staticmethod
+    def _h():
+        return 1 / np.sqrt(2) * np.array([[1, 1], [1, -1]])
```

Comment:
this method should just be deleted. all uses can be replaced by `HGate().to_matrix()`
where HGate is in the standard library of gates.

Reply:
done